ProjectSource/app-library/src/test/java/com/kayako/sdk/android/k5/messenger/homescreenpage/HomeScreenContainerPresenterTest.java

```diff
@@ -0,0 +1,57 @@
+package com.kayako.sdk.android.k5.messenger.homescreenpage;
+
+import junit.framework.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import static org.mockito.Mockito.verify;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.reflect.Whitebox;
+
+@RunWith(PowerMockRunner.class)
+public class HomeScreenContainerPresenterTest {
+
+    private HomeScreenContainerPresenter homeScreenContainerPresenter;
+
+    @Mock
+    private HomeScreenContainerContract.View view;
+
+    @Before
+    public void setUp() {
+        homeScreenContainerPresenter = new HomeScreenContainerPresenter(view);
+    }
+
+    @Test
+    public void OperationOnConversationButton() {
+        //Act
+        homeScreenContainerPresenter.initPage();
+        homeScreenContainerPresenter.onClickNewConversationButton();
+        final HomeScreenContainerContract.View viewLocal =
+                Whitebox.getInternalState(homeScreenContainerPresenter, "mView");
+
+        //Assert
+        Assert.assertEquals(view, viewLocal);
```

Comment:
⚠️  It could be a good idea to import the `static` assertions. This way the code becomes smaller and avoids repetition.  

**[More details](https://confluence.devfactory.com/x/MxNrEw)**
          
## The possible solution in this case

```JAVA
import static org.junit.Assert.assertEquals;
...
assertEquals(view, viewLocal); 
```
 

<a id="16ada6c0-54f5-11e8-9db9-b7d507f482ce" href="none" codereview-uuid-reserved-tag />